doc/user/content/get-started/quickstart.md

```diff
@@ -245,8 +245,12 @@ auction ended. As new auction and bid data appears, the query must be rerun to
 get up-to-date results.
 
 1. Using the [`CREATE VIEW`](/sql/create-view/) command, create a
-   [**view**](/concepts/views/ "Saved name/alias for a query") `winning_bids`
-   for the query that finds winning bids for auctions that have ended.
+   [**view**](/concepts/views/ "Saved name/alias for a query") to find the
```

Comment:
I think it makes more sense to add details about `DISTINCT ON` after the statement (like before). This might also be too low-level for a get started, so I'd keep it as a brief mention and just change the link to point to the new idiomatic SQL page, rather than the PostgreSQL documentation.

Reply:
Thanks for adding the suggestions ❤️